packages/web-components/src/components/va-search-input/va-search-input.tsx

```diff
@@ -402,6 +428,77 @@ export class VaSearchInput {
     const type = this.el.getAttribute('type') || 'text';
     /* eslint-enable i18next/no-literal-string */
 
+    if (uswds) {
+      const formClasses = classnames({
+        'usa-search': true,
+        'usa-search--big': big && !small,
+        'usa-search--small': small && !big,
+      });
+      return (
+        <Host onBlur={handleBlur}>
+          <form class={formClasses} role="search">
+            <label class="usa-sr-only" htmlFor="search-field">Search</label>
+            <input 
+              class="usa-input" 
+              id="search-field" 
+              name="search"
+              type="search"
+              ref={el => (this.inputRef = el as HTMLInputElement)}
+              aria-autocomplete={ariaAutoComplete}
+              aria-controls={ariaControls}
+              aria-expanded={ariaExpanded}
+              aria-haspopup={ariaHasPopup}
+              aria-label={label}
+              autocomplete="off"
+              onFocus={handleInputFocus}
+              onInput={handleInput}
+              onKeyDown={handleInputKeyDown}
+              role={role}
+              value={value}
+            />
+            <button 
+              class="usa-button" 
+              type="submit"
+              onClick={handleButtonClick}
+            >
+              {!small && <span class="usa-search__submit-text">Search</span>}
+              {small &&
+                <va-icon
```

Comment:
The USWDS markup has this being an `img` element but now that we have  the `va-icon` component, I thought it would be better to start using it for component composition.